Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -1175,6 +1175,24 @@ void ProgramShaderCache::UberShaderCompileWorkItem::Retrieve()
 
 void ProgramShaderCache::CreatePrerenderArrays(SharedContextData* data)
 {
+  // Create a framebuffer object to render into.
+  // This is because in EGL, and potentially GLX, we have a surfaceless context.
+  glGenTextures(1, &data->prerender_FBO_tex);
+  glBindTexture(GL_TEXTURE_2D_ARRAY, data->prerender_FBO_tex);
+  glTexImage3D(GL_TEXTURE_2D_ARRAY, 0, GL_RGBA, 1, 1, 1, 0, GL_RGBA, GL_UNSIGNED_BYTE, nullptr);
+  glTexParameteri(GL_TEXTURE_2D_ARRAY, GL_TEXTURE_MAX_LEVEL, 1);
+  glTexParameteri(GL_TEXTURE_2D_ARRAY, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
```

Comment:
You should set these parameters before the glTexImage3D call, especially max_level might affect the allocation